v3/src/models/data/formula.ts

```diff
@@ -5,29 +5,29 @@ import { getFormulaManager } from "../tiles/tile-environment"
 
 export const Formula = types.model("Formula", {
   id: types.optional(types.identifier, () => typedId("FORMULA")),
```

Comment:
Conventionally, the `typedId` prefix is four characters, so this would be `FORM`.

Reply:
Thanks, fixed.